gst/nnstreamer/tensor_converter/tensor_converter.c

```diff
@@ -824,15 +813,27 @@ gst_tensor_converter_chain (GstPad * pad, GstObject * parent, GstBuffer * buf)
       break;
     case _NNS_MEDIA_PLUGINS:
     {
+      GstTensorsConfig prev_config;
```

Comment:
recommended new_config
if update-caps is ok (line830), then update it. (self->tensors_config = mew_config;)
current patch does not handle this case.

Reply:
I changed it. Please check it one more time.